src/Notification/NotificationChannelRegistry.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Notification;
+
+class  NotificationChannelRegistry
```

Comment:
```suggestion
class NotificationChannelRegistry
```
Seems you don't have CS fixes configured in IDE